coffea/processor/executor.py

```diff
@@ -340,18 +349,69 @@ def work_queue_executor(items, function, accumulator, **kwargs):
         print-stdout : bool
             If true (default), print the standard output of work queue task on completion.
     """
-    try:
-        import work_queue  # noqa
-        import dill  # noqa
-        from .work_queue_tools import work_queue_main
-    except ImportError as e:
-        print("You must have Work Queue and dill installed to use work_queue_executor!")
-        raise e
 
-    return work_queue_main(items, function, accumulator, **kwargs)
+    # Standard executor options:
+    compression: Optional[int] = 9  # as recommended by lz4
+    retries: int = 2  # task executes at most 3 times
+    # wq executor options:
+    master_name: Optional[str] = None
+    port: Optional[int] = None
+    filepath: str = "."
+    events_total: Optional[int] = None
+    x509_proxy: Optional[str] = None
+    verbose: bool = False
+    print_stdout: bool = False
+    bar_format: str = "{desc:<14}{percentage:3.0f}%|{bar}{r_bar:<55}"
+    debug_log: Optional[str] = None
+    stats_log: Optional[str] = None
+    transactions_log: Optional[str] = None
+    password_file: Optional[str] = None
+    environment_file: Optional[str] = None
+    extra_input_files: List = field(default_factory=list)
+    wrapper: Optional[str] = shutil.which("python_package_run")
+    resource_monitor: bool = False
+    resources_mode: str = "fixed"
+    cores: Optional[int] = None
+    memory: Optional[int] = None
+    disk: Optional[int] = None
+    gpus: Optional[int] = None
+    chunks_per_accum: int = 10
+    chunks_accum_in_mem: int = 2
+    chunksize: int = 1024
```

Comment:
This is where I am not sure about. So in my understanding they should be part of the `Runner` class. This one just generates pre-chunked `WorkItems` which any executor just digests, no matter their chunksize. This is possible for alle executor, except maybe the `WorkQueue` one?

Reply:
Sounds like a good plan. `dynamic_chunksize` is an arg to both, so this should be the current state.

Reply:
As I thought about it last night, we don't even need the callback, as we already have the generator's .send method, so it will be just a matter to move some code to the chunk generator.